test/syntax.jl

```diff
@@ -1229,3 +1229,13 @@ end
 
 # issue #9972
 @test Meta.lower(@__MODULE__, :(f(;3))) == Expr(:error, "invalid keyword argument syntax \"3\"")
+
+@testset "underscore currying" begin
+    @test div(_, 3)(13) === 4
+    @test (_+1)(3) === 4
+    @test (_.re)(5+6im) === 5
+    @test (_[2])([7,8,9]) === 8
+    @test div.(10,_)([1,2,3,4]) == [10,5,3,2]
+    @test (_ .+ 1)([1,2,3,4]) == [2,3,4,5]
+    @Meta.lower(@__MODULE__, :(_ + _)) == Expr(:error, "only a single _ argument is allowed")
```

Comment:
Was this intended to be `@test Meta.lower` rather than `@Meta.lower`?

Reply:
whoops, will fix.

Reply:
I'm kind of amazed that CI had passed with that in there...

Reply:
Well there is a `@lower` in the `Meta` module so this would just become a `False`.

Reply:
I believe this hasn't yet been marked as resolved.